Doc/c-api/init.rst

```diff
@@ -221,6 +221,13 @@ to 1 and ``-bb`` sets :c:data:`Py_BytesWarningFlag` to 2.
    Set by the :option:`-v` option and the :envvar:`PYTHONVERBOSE` environment
    variable.
 
+.. c:var:: int Py_UTF8Mode
+
+   Enables the UTF-8 Mode.
+
+   If set to 1, enable the UTF-8 Mode.
+   If set to 0, disable the UTF-8 Mode.
+   If set to -1, look for the :option:`-x` option and the :envvar:`PYTHONUTF8` environment variable.
```

Comment:
Please mention that the variable is now deprecated: https://github.com/python/cpython/pull/93943